### DISCUSSION TITLE
No access to traefik dashboard on synology

### DISCUSSION THREAD

**Discussion by Bogey** [htpcBeginner/docker-traefik#159] [Support]
Hi, I was thrilled to find your guid on how to build a secure media server with docker on a synology.
I followed your guide to the t, but I cannot get the dashboard to show.
What suggestions do you have to get it up and running?
Thx!
Eddie

**Comment by SimpleHomelab** [comment]
What errors are you seeing? Check traffic logs as explained in the guide
